build.sbt

```diff
@@ -2,35 +2,33 @@ import ReleaseTransformations._
 
 name := "akka-xlsx"
 
-lazy val AkkaVersion        = "2.5.19"
-lazy val AkkaContribVersion = "0.9"
-lazy val AlpakkaXmlVersion  = "0.20"
+lazy val AkkaVersion        = "2.5.31"
+lazy val AkkaContribVersion = "0.10"
+lazy val AlpakkaXmlVersion  = "2.0.2"
 
-lazy val ScalatestVersion  = "3.0.5"
-lazy val ScalacheckVersion = "1.14.0"
+lazy val ScalatestVersion  = "3.0.9"
+lazy val ScalacheckVersion = "1.15.2"
```

Comment:
Again, both updated partially.

Reply:
That's just my laziness, but you're right, updated scalatest to newest version.